micro-rdk/src/esp32/pwm.rs

```diff
@@ -219,14 +219,31 @@ impl<'a> LedcTimerWrapper<'a> {
         if self.count > 0 {
             return Err(Esp32PwmError::OtherChannelsBindToTimer);
         }
+
         let id = {
```

Comment:
can you add the following comment:

```
// The configured clock source for the timer may not be able to achieve the target frequency,
	// We have to reconfigure the timer so the appropriate clock source for that frequency may be
	// selected. If no appropriate clock source exists the previous timer frequency
        // will be retained
```

Reply:
I have this commented further down, but it's not worded as well so I'll use this instead